kie-spring/pom.xml

```diff
@@ -258,6 +258,7 @@
           <systemPropertyVariables>
             <kie.maven.settings.custom>${project.build.testOutputDirectory}/kie-custom-settings.xml</kie.maven.settings.custom>
           </systemPropertyVariables>
+          <forkMode>always</forkMode>
```

Comment:
the reason why we have it here is that there are changes in transactionmanagerfactory - change of the impl. So we might be better of to add logic to reset the instance in that https://github.com/droolsjbpm/drools/blob/master/drools-persistence-jpa/src/main/java/org/drools/persistence/TransactionManagerFactory.java